p2p/mock/messageProcessorStub.go

```diff
@@ -11,7 +11,11 @@ type MessageProcessorStub struct {
 
 // ProcessReceivedMessage -
 func (mps *MessageProcessorStub) ProcessReceivedMessage(message p2p.MessageP2P, _ p2p.PeerID) error {
-	return mps.ProcessMessageCalled(message)
+	if mps.ProcessMessageCalled != nil {
+		return mps.ProcessMessageCalled(message)
```

Comment:
You can apply to the mock method the same signature as the ProcessReceivedMessage, with both parameters, as if someone will use this mock to gain the expected behavior

Reply:
ok

Reply:
done